<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QHeaderView::section,QTableCornerButton:section
{
    text-align:center;
    padding:3px;
    margin:0px;
    color:#DCDCDC;
    border:1px solid #242424;
    border-left-width:0px;
    border-right-width:1px;
    border-top-width:0px;
     border-bottom-width:1px;
    background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 54, 61);</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>1920</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 54, 61);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>425</y>
       <width>503</width>
       <height>225</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(17, 134, 243);
border-radius: 13px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labSuccess">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labHelmet">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>安全帽</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>当天比对</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>陌生人</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labMo">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>总人数</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labCoat">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 13px;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>反光衣</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labTotal">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>103</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(88, 96, 101);</string>
     </property>
     <widget class="QLabel" name="labTime">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>39</y>
        <width>422</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>125</y>
       <width>1020</width>
       <height>144</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>18</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
border-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));
font: 75 18pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(241, 100, 30);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>中建三局一公司建筑设计院</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>745</y>
       <width>1025</width>
       <height>505</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(80, 83, 87);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>298</y>
       <width>503</width>
       <height>352</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(86, 86, 86);
border-radius: 13px;</string>
     </property>
     <widget class="QLabel" name="labTemp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>258</width>
        <height>352</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 13px;  border:2px solidl;
border-color: rgb(255, 255, 255);
background-color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labGongZ">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>工种：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labName">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labCompany">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>公司：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labGroup">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>班组：</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>298</y>
       <width>503</width>
       <height>113</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 13px;
background-color: rgb(86, 86, 86);</string>
     </property>
     <property name="text">
      <string>新江湾-FD地块商办项目</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>672</y>
       <width>1025</width>
       <height>578</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 71, 78);
border-radius: 7px;</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>28</y>
        <width>174</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;Source Code Pro&quot;;</string>
      </property>
      <property name="text">
       <string>AI魔屏</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>1301</y>
       <width>1025</width>
       <height>578</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 71, 78);
border-radius: 7px;</string>
     </property>
     <widget class="QPushButton" name="btnPlate">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Source Code Pro&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>平台</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRecord">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Source Code Pro&quot;;
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetting">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Source Code Pro&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>75</y>
        <width>1025</width>
        <height>503</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(80, 83, 87);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>29</y>
        <width>236</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>中建三局—公司</string>
      </property>
     </widget>
    </widget>
    <zorder>widget_2</zorder>
    <zorder>widgetTop</zorder>
    <zorder>label_3</zorder>
    <zorder>widget_11</zorder>
    <zorder>label_4</zorder>
    <zorder>widget_4</zorder>
    <zorder>label</zorder>
    <zorder>widget_5</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
